### ISSUE TITLE
Invalid option: 1 when using fw()->theme->get_options()

### ISSUE THREAD

**Issue by maiakhoa** [ThemeFuse/Unyson#277]
I have two options files in theme options folder that showed on shortcode option by using fw()->theme->get_options('file-a').
For example : file-a and file-b.

when loading the dialog of shortcode, it occur the error below : 
Invalid option: 1 
And just one file is loaded.

**Comment by moldcraft** [comment]
Please show your file-a options array.

**Comment by maiakhoa** [comment]
This one for example : 

$options = array(
    'link' => array(
        'type'  => 'multi-picker',
        'label' => false,
        'desc'  => false,
        'picker' => array(
            'category_options' => array(
                'label'   => __('Link', 'fw'),
                'desc'    => __('Do you want to apply a link ?', 'fw'),
                'type'    => 'select',
                'value'   => 'no_link',
                'choices' => array(),
            ),
        ),
        'choices' => $choices,
        'show_borders' => false,
    ),
    'target' => array(
        'type'  => 'switch',
        'label'   => __( 'Open Link in New Window', 'fw' ),
        'desc'    => __( 'Select here if you want to open the linked page in a new window', 'fw' ),
        'value'   => '_self',
        'right-choice' => array(
            'value' => '_self',
            'label' => __('No', 'fw'),
        ),
        'left-choice' => array(
            'value' => '_blank',
            'label' => __('Yes', 'fw'),
        ),
    ),
);

**Comment by moldcraft** [comment]
What's in `$choices` variable?

**Comment by maiakhoa** [comment]
sorry It's just an array of multi-picker

**Comment by moldcraft** [comment]
Also give me please the full error message, I don't think it's only `Invalid option: 1`

**Comment by maiakhoa** [comment]
[23-Feb-2015 15:27:43 UTC] PHP Warning:  Invalid option: 1 in /Volumes/DATA/Working/mamp-htdocs/projectx/wp-content/themes/projectx/framework/helpers/general.php on line 704
[23-Feb-2015 15:27:43 UTC] PHP Stack trace:
[23-Feb-2015 15:27:43 UTC] PHP   1. {main}() /Volumes/DATA/Working/mamp-htdocs/projectx/wp-admin/admin-ajax.php:0
[23-Feb-2015 15:27:43 UTC] PHP   2. do_action($tag = 'wp_ajax_fw_backend_options_render', $arg = *uninitialized*) /Volumes/DATA/Working/mamp-htdocs/projectx/wp-admin/admin-ajax.php:85

**Comment by moldcraft** [comment]
Can you send me your theme (you can remove images and styles) to `unyson[at]email.com` so I can find the problem and ask this question faster?

**Comment by maiakhoa** [comment]
Hi @moldcraft 
I sent to you the theme. You could investigate with button shortcode

**Comment by maiakhoa** [comment]
Thank @moldcraft 
It's working fine with your change

**Comment by moldcraft** [comment]
@maiakhoa Thank you to
